servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/NettyChannelPublisher.java

```diff
@@ -69,6 +75,10 @@ protected void handleSubscribe(Subscriber<? super T> nextSubscriber) {
         }
     }
 
+    void readWriteObserver(@Nullable final NonMultiplexedObserver readWriteObserver) {
+        this.readWriteObserver = readWriteObserver;
```

Comment:
If you do end up doing this, clarify and verify whether this should only be accessed/assigned from the eventloop